ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -467,6 +467,81 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: '@yearly'
+  decorate: true
+  interval: ""
+  name: release-openshift-ocp-installer-console-aws-4.2
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-aws-cluster-profile
+      - --target=e2e-aws
+      - --template=/usr/local/e2e-aws
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:v4.2
+      - name: BRANCH
+        value: "4.2"
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)-console
```

Comment:
Looking at this again, we actually need it to look like:

```
- as: e2e-$(CLUSTER_TYPE)-console
  commands: ./test-prow-e2e.sh e2e
  openshift_installer_console:
    cluster_profile: "$(CLUSTER_TYPE)"
```

/hold cancel
/lgtm cancel